Example/Podfile

```
@@ -1,11 +1,12 @@
-platform :ios, '11.0'
+platform :ios, '12.0'
 use_frameworks!
 
 target 'TrustSDK_Example' do
   project 'TrustSDK'
   pod 'TrustSDK/Wallet', :path => '../'
   pod 'SwiftLint'
   pod 'CryptoSwift'
+  pod 'BigInt'
```

```suggestion
  pod 'BigInt'
```

this wasn't required before?